text/0550-macro-future-proofing.md

```diff
@@ -74,31 +74,36 @@ The algorithm for recognizing valid matchers `M` follows. Note that a matcher
 is merely a token tree. A "simple NT" is an NT without repetitions. That is,
 `$foo:ty` is a simple NT but `$($foo:ty)+` is not. `FOLLOW(NT)` is the set of
 allowed tokens for the given NT's fragment specifier, and is defined below.
```

Comment:
Add a sentence "A "complex NT" is an NT that is not simple.  (This way someone who does a search for "complex" in your document will find that definition.